python/arcticdb/adapters/s3_library_adapter.py

```diff
@@ -47,7 +47,7 @@ class ParsedQuery:
     # DEPRECATED - see https://github.com/man-group/ArcticDB/pull/833
     force_uri_lib_config: Optional[bool] = True
 
-    # winhttp is used as s3 backend support on Winodws by default; winhttp itself mainatains ca cert.
+    # winhttp is used as s3 backend support on Windows by default; winhttp itself mainatains ca cert.
```

Comment:
typo `mainatains`

Reply:
Wow this is a cheeky. Thanks for spotting!